app/assets/stylesheets/default_theme/table.scss

```diff
@@ -47,7 +47,8 @@
     }
   }
 
-  td {
+  td,
+  > tbody > tr > td {
```

Comment:
it needs to be more specific to override the default bootstrap styles, which are applied using the same rule (table>tbody>tr>td)

i included the more generic td selector to cover lazy/simple tables (tables without tbody)